packages/specification/schemas/components/_fragments/type.yaml

```diff
@@ -0,0 +1,21 @@
+components:
+  schemas:
+    OrganizationRole:
```

Comment:
The name of the entity is just "type" rather than "OrganizationRole;" could we please amend this to reduce ambiguity?

Reply:
This is just an internal identifier for when we reference this fragment.